dali/tensorflow/daliop.cc

```diff
@@ -147,8 +157,12 @@ class DaliOp : public tf::OpKernel {
 #endif
     LOG_LINE << "Pipeline created\n";
     LOG_LINE << "Prefetching...\n";
-    for (int i = 0; i < prefetch_queue_depth_; ++i) {
-      TF_DALI_CALL(daliRun(&pipe_handle_));
+    if (!exec_separated) {
+      TF_DALI_CALL(daliPrefetchUniform(&pipe_handle_, prefetch_queue_depth_));
+    } else {
+      TF_DALI_CALL(daliPrefetchSeparate(&pipe_handle_,
+                                        cpu_prefetch_queue_depth,
+                                        prefetch_queue_depth_));
```

Comment:
shouldn't it be gpu prefetch queue depth?

Reply:
It is the same, I am reusing the variable
I think that it makes sense because both denote the Pipeline GPU output queuedepth

Reply:
ok